### PR TITLE
Add some configs in tidb-lightning-configuration.md

### PR DESCRIPTION
**Pull request by sunzhaoyang**
fix https://github.com/pingcap/tidb/issues/31633
fix https://github.com/pingcap/docs/issues/7401#event-5901407658

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-lightning/tidb-lightning-configuration.md [hunk 1]
```diff
@@ -214,6 +214,26 @@ backslash-escape = true
 # 如果有行以分隔符结尾，删除尾部分隔符。
 trim-last-separator = false
 
+# [[mydumper.files]]
+# # 解析 parquet 文件所需的表达式
```
Comment (sunzhaoyang) [inline tidb-lightning/tidb-lightning-configuration.md:218]:
```suggestion
# # 解析aws aurora parquet 文件所需的表达式
```

File: tidb-lightning/tidb-lightning-configuration.md [hunk 2]
```diff
@@ -214,6 +214,26 @@ backslash-escape = true
 # 如果有行以分隔符结尾，删除尾部分隔符。
 trim-last-separator = false
 
+# [[mydumper.files]]
+# # 解析 aws aurora parquet 文件所需的表达式
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-configuration.md:218]:
```suggestion
# 解析 aws aurora parquet 文件所需的表达式
```

File: tidb-lightning/tidb-lightning-configuration.md [hunk 3]
```diff
@@ -214,6 +214,26 @@ backslash-escape = true
...
+# pattern = '(?i)^(?:[^/]*/)*([a-z0-9_]+)\.([a-z0-9_]+)/(?:[^/]*/)*(?:[a-z0-9\-_.]+\.(parquet))$'
+# schema = '$1'
+# table = '$2'
+# type = '$3'
+# 
+# # 设置分库分表合并规则，将 my_db1 中的 table1、table2 两个表,以及 my_db2 中的 table3、table4 两个表，共计 2 个数据库中的 4 个表都导入到目的数据库 my_db 中的 table5 表中。
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-configuration.md:224]:
```suggestion
# 设置分库分表合并规则，将 my_db1 中的 table1、table2 两个表，以及 my_db2 中的 table3、table4 两个表，共计 2 个数据库中的 4 个表都导入到目的数据库 my_db 中的 table5 表中。
```

**Review by hfxsd — APPROVED** [review]
Rest LGTM

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Review by kennytm — APPROVED** [review]

**Pull request merged**